app/models/concerns/search_cache.rb

```diff
@@ -17,7 +17,7 @@ def searchable_values_sql
   end
 
   def set_tsvector(value, weight)
-    "setweight(to_tsvector('spanish', coalesce(#{quote(value)}, '')), #{quote(weight)})"
+    "setweight(to_tsvector('#{dictionary_for_searches}', coalesce(#{quote(value)}, '')), #{quote(weight)})"
```

Comment:
this call to `dictionary_for_searches` is erroring and causing most of the [specs to fail](https://travis-ci.org/consul/consul/builds/96036595)